web/api/v1/api.go

```diff
@@ -147,6 +148,8 @@ func (api *API) Register(r *route.Router) {
 
 	r.Get("/targets", instr("targets", api.targets))
 	r.Get("/alertmanagers", instr("alertmanagers", api.alertmanagers))
+
+	r.Get("/read", prometheus.InstrumentHandler("read", http.HandlerFunc(api.remoteRead)))
```

Comment:
This isn't normal rest though, I'd be inclined to put it on a non-versioned endpoint for now and put it in the proper place once we've finalized the format (which basically means once we're ready to switch it to gRPC).